Server/src/main/kotlin/net/horizonsend/ion/server/listeners/bukkit/PlayerPickUpItemListener.kt

```diff
@@ -0,0 +1,31 @@
+package net.horizonsend.ion.server.listeners.bukkit
+
+import net.horizonsend.ion.common.database.Achievement
+import net.horizonsend.ion.server.utilities.ionCore
+import net.horizonsend.ion.server.utilities.rewardAchievement
+import net.starlegacy.feature.misc.CustomItems
+import org.bukkit.event.EventHandler
+import org.bukkit.event.EventPriority
+import org.bukkit.event.Listener
+import org.bukkit.event.player.PlayerAttemptPickupItemEvent
+
+@Suppress("unused")
+class PlayerPickUpItemListener : Listener {
+	@EventHandler(priority = EventPriority.MONITOR)
+	fun onPlayerPickUpItem(event: PlayerAttemptPickupItemEvent){
+		ionCore {
+		if (event.item == CustomItems.MINERAL_TITANIUM.singleItem()){
```

Comment:
use a `when` block, also learn how indentation works

Reply:
it autoindents but ok